frontend/components/TooltipWrapper/TooltipWrapper.tsx

```diff
@@ -85,6 +91,7 @@ const TooltipWrapper = ({
           disableStyleInjection
           clickable={clickable}
           offset={tipOffset}
+          positionStrategy={fixedPositionStrategy ? "fixed" : undefined}
```

Comment:
instead of this can we set `fixedPositionStrategy` to a default value in the params on lie 59? `fixedPositionStrategy = false`.

also the accepted values seem to be "fixed" and "absolute", not undefined.

Reply:
@ghernandez345 sure we can do that. `ReactTooltip5` internally defaults to `"absolute"`, so the thinking with passing in `undefined` was to say "let the component perform its default behavior." Changing `undefined` her to `"absolute"` will result in the same behavior, just has the connotation that we are moving that logic up a level to this component, which also makes sense.